### ISSUE TITLE
Form fields only display single error

### ISSUE THREAD

**Issue by TimMoser92** [TrestleAdmin/trestle#331]
Because of the following code, trestle admin only displays a single error even if mutliple errors exist for a single field. In my specific case there is a multi-select with two errors happening at once. The user will only be shown one at a time and has to fix them one after another.

https://github.com/TrestleAdmin/trestle/blob/6cbe886d2739a1533cb18da60883f5fa500dce11/lib/trestle/form/fields/form_group.rb#L36-L40

I would suggest changing `errors.first` and iterate over all errors, sth. like this:

```ruby
 def error_message 
   content_tag(:p, class: "invalid-feedback") do 
     errors.map do |error|
       safe_join([icon("fa fa-warning"), error], " ")
     end.safe_join("\n") 
   end 
 end
```

**Comment by spohlenz** [comment]
Nice suggestion, thank you. This was a shortcut I took early on, but it was easy enough to do this properly now.

Fixed in 21e4a18c52b4d2144bbd029e564419d0813bcddf.

**Comment by TimMoser92** [comment]
Thank you for the quick response and action 🙂 👍
